src/internal-links/handler.js

```diff
@@ -156,9 +157,12 @@ export async function convertToOpportunity(auditUrl, auditData, context) {
       type: 'CONTENT_UPDATE',
       rank: entry.traffic_domain,
       data: {
-        ...entry,
-        /* code commented until implementation of suggested links. TODO: implement suggestions, https://jira.corp.adobe.com/browse/SITES-26545 */
-        // suggestedLink: 'some suggestion here',
+        title: entry.title,
```

Comment:
need to update the OpenAPI schema separately (and approve).

Reply:
This [PR](https://github.com/adobe/spacecat-api-service/pull/713/files#diff-c22e97ef49f28d57b260ed547e238893e7086ad2c109c4d4fcff963f870e776bR1670) already has schema updates, its in review 
<img width="919" alt="image" src="https://github.com/user-attachments/assets/6c09325f-8130-4b0d-9e4b-1cb29cc038fd" />